TestColortoPairNumber.c

```diff
@@ -0,0 +1,69 @@
+#include "ColorPair.h"
+
+/**
+ ***************************************************************************************************
+ * Function Name: ColorPairToString. 
+ * 
+ * Function Description: Function to map the colorPair derived pairNumber via GetColorFromPairNumber() to colorPairNames.
+ *
+ * \param  colorPair:colorPair which comprises of majorColor and minorColor.
+ *         buffer: buffer to hold the colorPairNames.
+ *         
+ * \return  void
+ * \retval  None
+ ***************************************************************************************************
+ */
+
+void ColorPairToString(const ColorPair* colorPair, char* buffer) {
+    sprintf(buffer, "%s %s",
+        MajorColorNames[colorPair->majorColor],
+        MinorColorNames[colorPair->minorColor]);
+}
+
+/**
+ ***************************************************************************************************
+ * Function Name: GetColorFromPairNumber. 
+ * 
+ * Function Description: Calculate majorColor and minorColor from pairNumber.
+ *
+ * \param  pairNumber: pairNumber.
+ *                  
+ * \return  ColorPair stucture
+ * \retval  colorPair

```

Comment:
Avoid comments that dont add value beyond the code. What does retval colorPair add, which is not obvious in code?
Or else you could retain it and explain what the significance of the retval would be. E.g., what would it be, if the pairNumber is beyond the limit?